client/src/components/DisplayTable/DisplayTable.scss

```diff
@@ -63,10 +63,6 @@ table.display-table > thead > tr {
 
     color: $textLightColor;
 
-    &.center-text {
-      vertical-align: middle;
-    }
```

Comment:
Oh, sorry, one more weird thing here, didn't see this my self. This specific\* selector against `center-text` seems to be extending the `center-text` style to also center vertically. A little weird, but find if the `&` expands to something specific enough that this only applies here. Should probably leave this, just with `center-text` updated to `text-center`.

\* `&` is scss shorthand for whatever the selector one level up in the nesting is, so this is probably only applies to some specific `DisplayTable` html